packages/serum/src/market.ts

```diff
@@ -29,6 +29,8 @@ import {
   WRAPPED_SOL_MINT,
 } from './token-instructions';
 import { getLayoutVersion } from './tokens_and_markets';
+import { Token, ASSOCIATED_TOKEN_PROGRAM_ID } from '@solana/spl-token';
+import { connect } from 'http2';
```

Comment:
Unused?

Reply:
absolutely right, weird the linter didn't catch this 🤔 actually a little worrying...

Reply:
Update, it's just classified as a warning right now. I'll fix that here, and then follow up in a separate PR to fix the rest.